_books/ion-1-1/src/whats_new.md

```diff
@@ -6,26 +6,27 @@ perspective.
 ## Motivation
 
 Ion 1.1 has been designed to address some of the trade-offs in Ion 1.0 to make it suitable for a wider range of
-applications. Ion 1.1 now makes length prefixing of containers optional, and makes the interning of symbolic tokens
-optional as well. This allows for applications that write data more than they read data or are constrained by the
-writer in some way to have more flexibility. Data density is another motivation. Certain encodings (e.g., timestamps,
-integers) have been made more compact and efficient, but more significantly, macros now enable applications to have very
+applications, giving greater greater representational choice and expressive power.
+Some applications want to optimize writes over reads, or are constrained by the writer in some way (e.g. it's prohibitively expensive to buffer an entire value before writing). Ion 1.1 now makes both length prefixing of containers and the interning of symbol tokens independently optional, granting such writers greater flexibility. 
+Data density is another motivation. Certain encodings (e.g., timestamps,
+integers) have been made more compact and efficient. More significantly, macros now enable applications to have very
 flexible interning of their data's structure. In aggregate, data transcoded from Ion 1.0 to Ion 1.1 should be more
-compact.
+compact and more efficient to both read and write.
 
 ## Backwards compatibility
 
 Ion 1.0 and Ion 1.1 share the same data model.
 Any data that can be represented in Ion 1.0 can also be represented with full fidelity in Ion 1.1 and vice-versa.
 This means that it is always possible to convert data from one version to the other without risk of data loss.
 
-Ion 1.1 readers must be able to understand both Ion 1.0 and Ion 1.1 data.
+Ion 1.1 readers should be able to understand both Ion 1.0 and Ion 1.1 data.
 
 The text encoding grammar of Ion 1.1 is a superset of Ion 1.0's text encoding grammar.
 Any Ion 1.0 text data can also be parsed by an Ion 1.1 text parser.
 However, because Ion 1.1 has a different system symbol table,
 symbol IDs in an Ion 1.0 stream do not always refer to the same text as the same symbol ID in an Ion 1.1 stream.
-(For example: in an Ion 1.0 stream, `$4` is always the text `"name"`. However, it may or may not be `"name"` in an Ion 1.1 stream.)
+> [!TIP]
+> For example: in an Ion 1.0 stream, `$4` is always the text `"name"`. However, it may or may not be `"name"` in an Ion 1.1 stream. It may instead be user symbol 4 if the user has chosen not to export the system symbols.
```

Comment:
It feels a bit weird to have just the example part in the `TIP` block. I think I'd make this whole aside into a `NOTE`.

> NOTE: ~However,~ Because Ion 1.1 has a different system symbol table, [....]
> For example, in an Ion 1.0 stream, `$4` [...]